### DISCUSSION TITLE
Apple App-Specific Password

### DISCUSSION THREAD

**Discussion by weiran** [steilerDev/icloud-photos-sync#180] [Ideas]
Is it possible to use App-specific passwords with this? I can only get auth working with my actual Apple password which I don't want to be in the clear in the ENV vars.

**Comment by steilerDev** [comment]
Unfortunately Apple does not allow access of the Photos API using app specific passwords - those only apply to a small subset of the iCloud services.

I'm simply re-using the API exposed by the webinterface, and this one requires a full MFA authentication.
